core/src/main/kotlin/org/evomaster/core/output/service/RestTestCaseWriter.kt

```diff
@@ -201,7 +201,7 @@ class RestTestCaseWriter : HttpWsTestCaseWriter {
     override fun handleVerbEndpoint(baseUrlOfSut: String, _call: HttpWsAction, lines: Lines) {
 
         val call = _call as RestCallAction
-        val verb = call.verb.name.lowercase(Locale.getDefault())
+        val verb = call.verb.name.lowercase(Locale.ENGLISH)
```

Comment:
looks like several places use `Locale.getDefault()`. can you fix all of the occurrencies in this PR? also, might be good to specify a `setDefault` in the main entry point (ie `Main`) to avoid possible issues in future